webpack.base.config.js

```diff
@@ -66,6 +66,7 @@ let baseConfig = {
       config: path.resolve(__dirname, 'config.js'),
       homepage: process.env.CUSTOM_HOMEPAGE === 'true' ? path.resolve(__dirname, 'custom/homepage.js') : ('util/homepage.js'),
       lbryinc: 'lbryinc/dist/bundle.es.js',
+      dgram: 'dgram-browserify',
```

Comment:
The UDP sockets aren't available on a local app just in browser, in order to still be able to analyse the sockets and find reliable Chromecasts on a local app (even though it's web based), I replaced the default datagram through the dgram-browserify package (https://www.npmjs.com/package/dgram-browserify). To avoid any mistakes I added this alias in order to let webpack know which package to use when dgram is called. Besides of the package I created there are no other package on LBRY which seem to use dgram.